.github/ISSUE_TEMPLATE/gutenberg-plugin-upgrade.md

```diff
@@ -31,7 +31,7 @@ As you complete the tasks in this list, please update the relevant lines with di
 - [ ] Install and activate [v#.#.#]-rc.1 on edge (######-code)
 - [ ] Install and activate [v#.#.#] on edge (######-code)
 - [ ] Activate [v#.#.#] on production (######-code)
-- [ ] Request [v#.#.#] AT upgrade (######-p2)
+- [ ] Release build of [v#.#.#] is available on intermediary repo to be pulled by AT (https://github.com/Automattic/gutenberg-dotcom-woa-releases/releases/tag/[v#.#.#])
```

Comment:
Is "Release" a verb here or a noun? If it's a verb (which it should ideally be as it's an action item), then the sentence doesn't really make sense and I would be confused by it.

If "Release" is a noun, then I think we should change it to be a verb.

Reply:
Thanks Donna! I basically wrote it as a noun, but yap verb makes more sense, rest of the instructions also begin with verb. I've updated it here https://github.com/Automattic/wp-calypso/pull/95345/commits/9db6cbffbef76e3c6e71a76e16dfb6880d01e26b

Now it says - Release the [v#.#.#] build to intermediary repo for AT to pull (https://github.com/Automattic/gutenberg-dotcom-woa-releases/releases/tag/[v#.#.#])